src/patent_fetcher.py

```diff
@@ -8,9 +8,9 @@
 from io import BytesIO
 from celery import Celery
 
-celery = Celery(__name__)
-celery.conf.broker_url = os.getenv("BROKER_URL")
-celery.conf.result_backend = os.getenv("BACKEND_URL")
+from src.flask_celery import make_celery
+
+celery = make_celery()
```

Comment:
Seems the celery on the left (deleted lines) was not working with the flask app?